syntax = "proto3";
package bcos.consensus;
message RawProposal
{
  // the index of the proposal
  int64 index = 1;
  // the hash of the proposal
  bytes hash = 2;
  // the proposal data, optional
  bytes data = 3;
  // the proposal signature, optional
  bytes signature = 4;
  bytes extraData = 5;
  int64 sealerId = 6;
  bool systemProposal = 7;
};